cmd/status.go

```diff
@@ -0,0 +1,41 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package cmd
+
+import (
+	"os"
+
+	"github.com/dapr/cli/pkg/kubernetes"
+	"github.com/dapr/cli/pkg/print"
+	"github.com/dapr/cli/utils"
+	"github.com/gocarina/gocsv"
+	"github.com/spf13/cobra"
+)
+
+var StatusCmd = &cobra.Command{
+	Use:   "status",
+	Short: "Shows status and health of the Dapr control plane on Kubernetes",
```

Comment:
The command will not be kubernetes specific (its for now) so remove kubernetes from it.
placement is for local development as well, so allow it.

Reply:
We have kubernetes specific flags for many other commands. If we want to remove the flag entirely it should be done on a PR with that goal.

Reply:
yes, that can be done but this PR can do the right thing and not add Kubernetes here.
The text can just read:
"Shows status and health of the Dapr control plane."

Reply:
But that won't be the case: This PR is about Dapr control plane status on Kubernetes.

Reply:
Currently, Kubernetes is the only platform where the control plane is running.
If there are other platforms in the future, we will enable platform specific flags for those.

Reply:
yes, i understand that, and at that time, you would change this text to be:
"Shows status and health of the Dapr control plane."
You will not be adding all the platforms to it to be: "Shows status and health of the Dapr control plane on Kubernetes, Service Fabric, IoT Edge". The platform would be enforced by the flag.
So I am just asking the text to be and nothing more here:
"Shows status and health of the Dapr control plane."

Reply:
Ah ok I thought you meant something different for some reason.

Reply:
Done

Reply:
My feedback
1)  We have now invented two terms - Dapr system services and Dapr control plane and we should stick with one.
2) Does this work for dapr init -k where this is deployed to Kubernetes default namespace?
3) Will there be a PR to do this command on self hosted?

Reply:
I think we should stick with Control Plane as its where the rest of the ecosystem is in terms of terminology.

Currently we only have the Control Plane for Kubernetes: Injector and Operator are k8s only, placement is not really part of the control plane, and for sentry and placement the amount of details (like healthy/unhealthy) you can get in self-hosted mode are very scarce as opposed to Kubernetes which is designed to supply that info.

Reply:
And yes, this also works with `dapr init -k`.